react-native.config.js

```diff
@@ -15,7 +16,8 @@ const project = (() => {
         ),
       },
       ios: {
-        project: iosProjectPath('example/ios'),
+        sourceDir: 'example/ios',
```

Comment:
Turns out, this is only valid in `@react-native-community/cli` 8.0+, and invalid otherwise. I'm going to pull this into `react-native-test-app` to simplify a bit.

Reply:
Shall we wait for changes in upstream, before merging this then?

Reply:
Yeah, I have a PR here: https://github.com/microsoft/react-native-test-app/pull/1293

Need to figure out why it's failing on Windows and my Windows box decided to stop booting up. I'm currently re-installing it… 😩

Reply:
I recently dusted out an old laptop with Win7 and I wanted to go with Win10. Turns out, it's so old, it cannot even install Win10 😄